src/main/scala/org/broadinstitute/dsde/workbench/gpalloc/monitor/ProjectCreationMonitor.scala

```diff
@@ -69,15 +72,15 @@ class ProjectCreationMonitor(projectName: String,
     case Fail(failedOps) =>
       logger.error(s"Creation of new project $projectName failed. These opids died: ${failedOps.map(op => s"id: ${op.operationId}, error: ${op.errorMessage}").mkString(", ")}")
       cleanupOnError()
-      stop(self)
+      throw new MonitorFailedException(projectName) //throw to supervisor. should be the last thing because it breaks control flow!
 
     //stop because something (probably google polling) throw an exception
     case Failure(throwable) =>
       val stackTrace = new StringWriter
       throwable.printStackTrace(new PrintWriter(new StringWriter))
       logger.error(s"Creation of new project $projectName failed because of an exception: ${throwable.getMessage} \n${stackTrace.toString}")
       cleanupOnError()
-      stop(self)
+      throw new MonitorFailedException(projectName) //throw to supervisor. should be the last thing because it breaks control flow!
```

Comment:
this case does cover things like 429 ratelimit errors, which might benefit from restarting the monitor rather than replacing the project. given that we generally don't pay attention to the errors gpalloc throws, i think that unnecessarily recreating the project (ie this behaviour, which will likely terminate once some project creation succeeds) is probably safer than restarting the monitor (which may continue to fail forever if it's not a transient error).